content/articles/wavelet-transform-based-denoising-of-images-using-matlab/index.md

```diff
@@ -0,0 +1,141 @@
+### Wavelet-Based Denoising of images using Matlab
+### Introduction
+wavelet analysis is a method of analysis that uses the time-frequency analysis method to select an appropriate frequency band based on the 
+characteristics of the signal. Getting rid of the noise that may contaminate the image during transfer is a challenge. Several algorithms 
+have been written and some discussed but it remains to be a challenge. We will discuss the currently available method, that is, wavelet-based 
+denoising of images, which is the currently efficient method. Signal transmission over a distance may be contaminated with noise. The objective 
+here is to remove the noise from the image using the wavelet technique. 
+The signal acquires the noise through the additive method and it has the form below.
+f'(x,y) = f(x,y) + n(x,y) where f'(x,y) is the contaminated signal, f(x,y) is the original signal and n(x,y) is the noise signal. The basic assumption of noise signal n(x,y) for the proposed scheme are;
+- Noise is additive
+- Noise is a random signal(white gaussian with zero mean value)
+- Noise has a high frequency.
+The objective here is to remove the noise n(x,y) from the noisy image f'(x,y) using the wavelet technique.
+
+### Prerequisite
+1. [Matlab](https://www.mathworks.com/downloads/) installed.
+2. A proper understanding of [Matlab](https://www.section.io/  engineering-education/getting-started-with-matlab/) language.
+
+### Wavelet-Based Denoising scheme
+The denoising scheme involves passing the signal through decomposition to be decomposed into various wavelet co-efficient 
+using discrete wavelet transform(DWT). The coefficients are then thresholded and then collected together. image thresholding 
+is the separation of foreground and background. It is a method of image segmentation and it isolates grayscale images by 
+converting them to binary images. using the inverse discrete wavelet transform(IDWT) to get the denoised image. Finding threshold 
+labels, we have various methods such as universal threshold, Bayes, SURE, MinMax, e.t.c.
```

Comment:
Explain in-detail about:
- discrete wavelet transform(DWT)
- Grayscale images
- Binary images
- inverse discrete wavelet transform(IDWT) - add external references